src/test/shell/integration/configured_query_test.sh

```diff
@@ -90,6 +90,20 @@ EOF
  assert_contains "name: \"//$pkg:japanese\"" output
 }
 
+function test_basic_query_output_labelkind() {
+  local -r pkg=$FUNCNAME
+  mkdir -p $pkg
+  cat > $pkg/BUILD <<EOF
+sh_library(name='maple', deps=[':japanese'])
+sh_library(name='japanese')
+EOF
+
+ bazel cquery --output=label_kind "deps(//$pkg:maple)" > output 2>"$TEST_log" || fail "Expected success"
+
+ assert_contains "name: \"//$pkg:maple\"" output
```

Comment:
99% of the way there. But this and the line below are going to fail because it's not actually outputting `//$pkg:maple` now, which is what the `label_kind` option does, right?

From the red test results attached to this PR:

https://storage.googleapis.com/bazel-untrusted-buildkite-artifacts/270d91f7-5738-4269-a656-c3c98f7c3550/src/test/shell/integration/configured_query_test/attempt_1.log

You can see the actual output looks like:

    sh_library rule 
    platform rule
    constraint_value rule
    ...

Reply:
Yea the task what we wanted is being achieved before only. We dont require these two lines, right?